components/Forms/Radio.tsx

```diff
@@ -45,17 +45,21 @@ export const Radio: React.VFC<InputProps> = observer((props) => {
             dangerouslySetInnerHTML={{ __html: label }}
           ></span>
         </label>
-        <span>
+        {/* <span>
```

Comment:
if this is not needed can we please remove this

Reply:
Yes, I just reverted this change, I will create a separate PR to address that change